test/launchdarkly_flutter_test.dart

```diff
@@ -85,10 +85,16 @@ void main() {
     expect(await launchdarklyFlutter.init('MOBILE_KEY', null), true);
   });
 
-  test('init with all arguments', () async {
+  test('init with mobile key and user', () async {
     expect(await launchdarklyFlutter.init('MOBILE_KEY', 'USER_ID'), true);
   });
 
+  test('init with all arguments', () async {
```

Comment:
Can we have a few more tests with edge cases, please?
- custom with other types, not only String, as in this case. For instance, also:  Number, Boolean and also `null`.

Reply:
I've added these types into the test. As for me, it makes sense to have them in one test: `null` case is covered by other cases, and `customValues` map covers all supported values.